src/api.ts

```diff
@@ -145,7 +145,7 @@ export async function getToken(
       throw new Error(error.response.data ?? "Authentication error.");
     }
 
-    process.stderr?.write(`ERROR received from ${requestAddress}: ${error}\n`);
+    console.error(`ERROR received from ${requestAddress}: ${error}\n`);
```

Comment:
if you logged in with a bad password, the returned error would be `process not defined` because that would be the exception thrown here